.github/workflows/release-sponsored-txs-docker.yml

```
@@ -0,0 +1,55 @@
+# This job builds and publishes a docker image for the sponsored transactions dApp to
+# the dockerhub image repository.
+name: Create and publish a Docker image for the sponsored transactions dApp.
+
+on:
+  workflow_dispatch: # allows manual trigger
+
+  push:
+    tags:
+      # The versioning is of the format: `frontend_version-backend_version`.
+      - 'sponsored-transactions/*.*.*-*.*.*'
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: dapp-sponsored-txs
```

Can we use the same naming convention to the above `tags`

```suggestion
  IMAGE_NAME: dapp-sponsored-transactions
```

---

The repository name already exists as `txs`.